lightning stops building

I don't know why but our current master stopped building even in a clean environment and after running `make clean` the issue seems to be `tool/mockup.sh: line 11:`		;;`

I have no clue at all why it stopped working and I am not that deep into the build process of c code.

so even in a completely fresh git clone I run into this issue after `./configure` and `make`

operating system is mac-os. I can later check on a linux machine but I guess if the problem was there too you would already know.

```
tools/generate-wire.py --page impl tools/test/gen_test.h test_type < tools/test/test_cases > 
tools/test/gen_test.c:
tools/mockup.sh: line 11: syntax error near unexpected token `;;'
tools/mockup.sh: line 11: `		;;'
make: *** [tools/test/gen_test.c] Error 2
rm external/libwally-core/src/secp256k1/libsecp256k1.la
Renes-iMacReal:lightning rpickhardt$ 
``` 

## git log
```
commit 0ae20399bd05adf8b6c1827359d9ae42e5f7376b (HEAD -> master, origin/master, origin/HEAD)
Author: trueptolemy <823220586@qq.com>
Date:   Sun Jul 28 21:24:57 2019 +0800

    CHANGELOG: check the bitcoind version when `setup_topology`

commit 130bf20516be815acc5034ba84b9c106ac6274a9
Author: trueptolemy <823220586@qq.com>
Date:   Sun Jul 28 01:58:27 2019 +0800

    lightningd: check bitcoind version when `setup_topology`
```

actually on linux I have 

```
tools/generate-wire.py --page header common/gen_status_wire.h status < common/status_wire.csv > common/gen_status_wire.h
Traceback (most recent call last):
  File "tools/generate-wire.py", line 24, in <module>
    from mako.template import Template
ModuleNotFoundError: No module named 'mako'
common/Makefile:83: recipe for target 'common/gen_status_wire.h' failed
make: *** [common/gen_status_wire.h] Error 1
rm external/libwally-core/src/secp256k1/libsecp256k1.la
```
indicating that a pythong dependency for the tests is missing. was able to fix this with
`pip install mako` will open a PR in 1 minute in which I add the mako dependency to  `tests/requirements.txt`

--- a/tests/requirements.txt
+++ b/tests/requirements.txt
@@ -5,6 +5,7 @@ Flask==1.0.2
 cheroot==6.5.4
 ephemeral-port-reserve==1.1.0
 flaky==3.5.3
+mako==1.0.14
 pytest-benchmark==3.2.2
 pytest-forked==1.0.1
 pytest-xdist==1.26.0

